README.md

```diff
@@ -1,4 +1,5 @@
-# bloomrun
+![logo](./docs/imgs/bloomrun-banner.jpg)
```

Comment:
I would live the h1, because otherwise the markdown is not self-explanatory. I open READMEs in vim :).